### PR TITLE
fix: Disable view data option  in the agnostic table context menu

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,8 @@
     "sense": "nebula sense",
     "spec": "scriptappy-from-jsdoc -c ./spec-configs/props.conf.js",
     "test:unit": "aw -c aw.config.js",
-    "test:integration": "aw puppet --testExt '*.int.js' --glob 'test/integration/**/*.int.js'"
+    "test:integration": "aw puppet --testExt '*.int.js' --glob 'test/integration/**/*.int.js'",
+    "extension": "yarn build && cp -r ./dist ~/Qlik/Sense/Extensions"
```
Comment (haxxmaxx) [inline package.json:37]: Don't want to push this I think, it's very specific to developing against sense-client

Comment (LiKang6688) [reply package.json:37]: @haxxmaxx  I imagine that we will develop against sense-client quite often. Do you think so?

Comment (haxxmaxx) [reply package.json:37]: We will, but we should keep the references to private code to a minimum. i don't think this command  is applicable if you don't have access to sense-client

Comment (LiKang6688) [reply package.json:37]: Good point. Did not consider that. I will remove it.

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**